tests/modules/asset_groups/resources/utils.py

```diff
@@ -281,7 +281,7 @@ def read_asset_group_sighting_as_sighting(
         # startTime and locationId are only present in the _as_sighting endpoints,
         # since they are in the config of a standard AGS
         response_200={
-            'guid',
+            'id',
```

Comment:
Are we going with 'id' over 'guid' in all cases now?

Reply:
Nope, this may be a mistake, it evolved. Right this minute it should be both id and guid until Ben changes the frontend to be guid only. Having guid in some messages and id in others is just a pain that we don't need.